### PR TITLE
feat(dedicated): added gateway ip in network tile

### PR DESCRIPTION
**Pull request by vks-ovh**
added gateway ip in network tile for dedicated servers
 ref: MANAGER-9077

Signed-off-by: vikash singh <vikash.singh@corp.ovh.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-9077
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Added IPv4 gateway ip under network tile for dedicated servers

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html [hunk 1]
```diff
@@ -557,6 +557,19 @@
                     </oui-action-menu>
                 </oui-tile-definition>
 
+                <!-- Gateway IP-->
+                <oui-tile-definition
+                    term="{{:: 'server_configuration_gateway_ip' | translate }}"
+                >
+                    <oui-tile-description>
+                        <oui-clipboard
+                            class="ip-clipboard"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html:566]: Question: is the class "ip-clipboard" defined? is it required?

**Review by radireddy — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html (hunk 1 above)
Comment (vks-ovh) [reply packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html:566]: yes, the class is already defined and it is required as well.

**Review by JeremyDec — COMMENTED** [review]
Hello @vks-ovh, I'm ok with the changes, could you please amend your commit, and use present tense for the commit message? Following Git commits preferences, your commit message should be `feat(dedicated): add gateway ip in network tile`

Thanks

**Review by ghyenne — APPROVED** [review]

**Review by radireddy — APPROVED** [review]

**Review by mohammed-zahaf — DISMISSED** [review]

**Review by antleblanc — CHANGES_REQUESTED** [review]
https://github.com/ovh/manager/runs/6107914383

**Review by ghyenne — DISMISSED** [review]

**Review by JeremyDec — CHANGES_REQUESTED** [review]
@vks-ovh missing ticket reference in the commit. Can you please add it?

**Review by ghyenne — DISMISSED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Pull request merged**